cw-orch-daemon/src/env.rs

```diff
@@ -0,0 +1,133 @@
+//! This regroups all env variables used by cw-orch-daemon. It allows for easier documentation and env variable management
+//! This is used to import environment variables with safe names (and at a centralized location)
+//! To get the env variable parsed value, you can use
+//! ```rust,no_run
+//! use cw_orch_daemon::DaemonEnvVars;
+//! let env_variable = DaemonEnvVars::state_file();
+//! ```
+
+use std::{env, path::PathBuf, str::FromStr};
+
+use cosmwasm_std::StdError;
+
+const DEFAULT_TX_QUERY_RETRIES: usize = 50;
+
+pub const STATE_FILE_ENV_NAME: &str = "STATE_FILE";
+pub const GAS_BUFFER_ENV_NAME: &str = "CW_ORCH_GAS_BUFFER";
+pub const MIN_GAS_ENV_NAME: &str = "CW_ORCH_MIN_GAS";
+pub const MAX_TX_QUERIES_RETRY_ENV_NAME: &str = "CW_ORCH_MAX_TX_QUERY_RETRIES";
+pub const MIN_BLOCK_SPEED_ENV_NAME: &str = "CW_ORCH_MIN_BLOCK_SPEED";
+pub const DISABLE_WALLET_BALANCE_ASSERTION_ENV_NAME: &str =
```

Comment:
Maybe a nit but are we defaulting to having the env-var named as `disable` and basically defaulting to setting env-vars to true if they want to enable something (so default all our bool env vars to false) or do we change it to the env-vars always indicating the feature and having the env var toggle them. I.e. rename these "disable" env vars to "wallet_balance_assertion", "log_activation", etc. and having them set to a default value.